liquidation-monitor/src/agent.spec.js

```diff
@@ -48,14 +101,31 @@ jest.mock('forta-agent', () => ({
 }));
 
 const {
-  ethers, Finding, FindingType, FindingSeverity,
+  ethers, Finding, FindingType, FindingSeverity, TransactionEvent,
 } = require('forta-agent');
 const config = require('../bot-config.json');
 
 const {
-  provideInitialize, createAlert,
+  provideInitialize, provideHandleBlock, provideHandleTransaction,
 } = require('./agent');
 
+// Convert the string numbers to ethers.BigNumber
+/* eslint-disable new-cap */
+mockBtcPrice = new ethers.BigNumber.from(mockBtcPrice);
+mockEthPrice = new ethers.BigNumber.from(mockEthPrice);
+mockUsdcPrice = new ethers.BigNumber.from(mockUsdcPrice);
+mockCDecimals = new ethers.BigNumber.from(mockCDecimals);
+mockBtcDecimals = new ethers.BigNumber.from(mockBtcDecimals);
+mockEthDecimals = new ethers.BigNumber.from(mockEthDecimals);
+mockUsdcDecimals = new ethers.BigNumber.from(mockUsdcDecimals);
+mockBtcCollateralFactor = new ethers.BigNumber.from(mockBtcCollateralFactor);
+mockEthCollateralFactor = new ethers.BigNumber.from(mockEthCollateralFactor);
+mockUsdcCollateralFactor = new ethers.BigNumber.from(mockUsdcCollateralFactor);
+mockBtcCTokenRate = new ethers.BigNumber.from(mockBtcCTokenRate);
+mockEthCTokenRate = new ethers.BigNumber.from(mockEthCTokenRate);
+mockUsdcCTokenRate = new ethers.BigNumber.from(mockUsdcCTokenRate);
+const mockEthersZero = new ethers.BigNumber.from(0);
```

Comment:
It looks like the constructor cannot be called directly, so taking out the `new` keyword seems appropriate:
```suggestion
mockBtcPrice = ethers.BigNumber.from(mockBtcPrice);
mockEthPrice = ethers.BigNumber.from(mockEthPrice);
mockUsdcPrice = ethers.BigNumber.from(mockUsdcPrice);
mockCDecimals = ethers.BigNumber.from(mockCDecimals);
mockBtcDecimals = ethers.BigNumber.from(mockBtcDecimals);
mockEthDecimals = ethers.BigNumber.from(mockEthDecimals);
mockUsdcDecimals = ethers.BigNumber.from(mockUsdcDecimals);
mockBtcCollateralFactor = ethers.BigNumber.from(mockBtcCollateralFactor);
mockEthCollateralFactor = ethers.BigNumber.from(mockEthCollateralFactor);
mockUsdcCollateralFactor = ethers.BigNumber.from(mockUsdcCollateralFactor);
mockBtcCTokenRate = ethers.BigNumber.from(mockBtcCTokenRate);
mockEthCTokenRate = ethers.BigNumber.from(mockEthCTokenRate);
mockUsdcCTokenRate = ethers.BigNumber.from(mockUsdcCTokenRate);
const mockEthersZero = ethers.BigNumber.from(0);
```

Ref: https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--creating

Reply:
Fixed all instances of this in commit 7717368c93664a46254a77ad97ed46aebd8f3dcf